cpp/velox/compute/VeloxExecutionCtx.h

```diff
@@ -30,6 +30,8 @@
 
 namespace gluten {
 
+inline static const std::string kVeloxExecutionCtxKind{"velox"};
```

Comment:
Please add a comment to note that the flag should be exactly the same to Java-side `VeloxBackend#getName()`

Reply:
Or we can add a comment to the native registry API. Both is OK to me. This is to prevent the name mapping from being implicit to users.

Reply:
add a comment, thanks!